sktime/classifiers/proximity.py

```diff
@@ -0,0 +1,1084 @@
+                                                                49.6, 53.2, 56.8, 60.4,
+                                                                64, 67.6, 71.2, 74.8, 78.4, 82, 85.6, 89.2, 92.8, 96.4,
+                                                                100]
+                    },
+            ]
+    return param_pool
+
+
+class ProximityStump(Classifier):
+    '''
+    proximity tree classifier of depth 1 - in other words, a k=1 nearest neighbour classifier with neighbourhood limited
+    to x exemplar instances
+    ----
+    Parameters
+    ----
+    pick_exemplars_method : callable
+        Method to pick exemplars from a set of instances and class labels
+    param_perm : dict
+        a dictionary containing a distance measure and corresponding parameter
+    gain_method : callable
+        a method to calculate the gain of this split / stump
+    label_encoder : LabelEncoder
+        a label encoder, can be pre-populated
+    rand : numpy RandomState
+        a random state for sampling random numbers
+    ----
+    Attributes
+    ----
+    exemplar_instances : panda dataframe
+        the chosen exemplar instances
+    exemplar_class_labels : numpy 1d array
+        array of class labels corresponding to the exemplar instances
+        the exemplar instances class labels
+    remaining_instances : panda dataframe
+        the remaining instances after exemplars have been removed
+    remaining_class_labels : numpy 1d array
+        array of class labels corresponding to the exemplar instances
+        the remaining instances class labels after picking exemplars
+    branch_instances : list of panda dataframes
+        list of dataframes of instances, one for each child of this stump. I.e. if a stump splits into two children,
+        there will be a list of dataframes of length two. branch_instance[0] will contain all train instances
+        closest to exemplar 0, branch_instances[1] contains all train instances closest to exemplar 1,
+        etc. Exemplars are in the exemplar_instances variable
+    branch_class_labels: list of numpy 1d arrays
+        similar to branch_instances, but contains the class labels of the instances closest to each exemplar
+    distance_measure_param_perm: dict
+        parameters to pass to the distance measure method
+    distance_measure: callable
+        the distance measure to use for measure similarity between instances
+    gain: float
+        the gain of this stump
+    label_encoder : LabelEncoder
+        a label encoder, can be pre-populated
+    classes_ :
+        pointer to the label_encoder classes_
+    '''
+
+    __author__ = 'George Oastler (linkedin.com/goastler)'
+
+    def __init__(self,
+                 pick_exemplars_method = None,
+                 param_perm = None,
+                 gain_method = None,
+                 label_encoder = None,
+                 rand = np.random.RandomState):
```

Comment:
Yes, so here you could just have `None` as the default and then pass that to `check_random_state()`